zachjs-sv2v/build.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+set -x
+set -e
+
+STACK_BINARY="https://github.com/commercialhaskell/stack/releases/download/v2.5.1/stack-2.5.1-linux-x86_64-bin"
+
+# Pull Haskell stack binary
+# stack version from conda-forge forces us to use
```

Comment:
can we just force an older version of stack?

Reply:
Yes we can. Recipe updated.